notebooks/basic-usage.ipynb

```diff
@@ -0,0 +1,386 @@
+{
```

Comment:
<p>nitpick, what about using a number of samples other than 1000, just to get a more visual understanding of the meaning of each dimension?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc4/pull/190/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>